README.md

```diff
@@ -25,14 +27,40 @@ For more details about the UberJar please head over to the
 [How to Build AEM Projects using Apache Maven](https://helpx.adobe.com/experience-manager/6-4/sites/developing/using/ht-projects-maven.html) documentation page.
 
 ## System Requirements
-The latest version of the CIF Core Components, require the below minimum system requirements:
+The latest version of the AEM CIF Core Components, require the below minimum system requirements:
 
 | CIF Core Components | AEM 6.4 | AEM 6.5 | Magento | Java |
 |---------------------|---------|---------|---------|------|
 | 0.0.1               | 6.4.4.0 | 6.5.0   | 2.3.1   | 1.8  |
 
+Additionally, it is highly recommended to use this project together with the [AEM Commerce connector for Magento](https://github.com/adobe/commerce-cif-connector). This is no hard dependency but some authoring functionality will be limited without the AEM Commerce connector for Magento as product pickers, product assets view and consoles are not available.
+
 For a list of requirements for previous versions, see [Historical System Requirements](VERSIONS.md).
- 
+
+## CIF Magento GraphQL Configuration
+
+The AEM CIF Core Components connect to a Magento via GraphQL that has to be configured to access your Magento instance. Follow the steps below to configure the bundle: 
+
+1) Configure the generic GraphQL instance
+    * Go to http://localhost:4502/system/console/configMgr
+    * Look for _CIF GraphQL Client Configuration Factory_
+    * Create a child configuration
+        * Keep the `default` service identifier or set something custom. Make sure to use the same value in step 2) below.
+        * For _GraphQL Service URL_ enter the URL of your Magento GraphQL endpoint (usually `https://hostname/graphql`)
+        * If you use this project together with the [AEM Commerce connector for Magento](https://github.com/adobe/commerce-cif-connector) this has to be configured only once.
+
+2) Link your AEM site with the GraphQL client instance
+    * Go to http://localhost:4502/crx/de/index.jsp
+    * Navigate to the page root node e.g. /content/venia/jcr:content for the Venia sample page
+    * Add a property `cq:graphqlClient` with the value of the service identifier of step 1
+    
+3) Assign the Magento root category of to your website
+    * Go to AEM Sites console http://localhost:4502/sites.html/
+    * Navigate to the catalog page with your site structure e.g. /content/venia/us/en/products for the Venia sample page
+    * Select the catalog page and open page properties
+    * Select the Commerce tab
+    * Enter the root category of your Magento catalog the site should be assigned or use the category picker
```

Comment:
Sorry one more comment: maybe you should write "root category id", and I guess here the picker will not work without the connector, right? Maybe add an extra comment in parentheses?

Reply:
Changed the wording. The picker can, at the moment, only be used to select any category except the root category.